sdk/src/main/scala/com/horizen/SidechainSettings.scala

```diff
@@ -42,7 +42,8 @@ case class ForgerSettings(automaticForging: Boolean = false,
                           allowedForgersList: Seq[ForgerKeysData] = Seq())
 
 case class WalletSettings(seed: String,
-                          genesisSecrets: Seq[String])
+                          genesisSecrets: Seq[String],
+                          maxFee: Int = -1)
```

Comment:
A few points here:
1) It can be useful to set default value enabled for every node.
On mainchain it is:
```
//! -maxtxfee default
static const CAmount DEFAULT_TRANSACTION_MAXFEE = 0.1 * COIN;
```
2) let's use `maxTxFee` name as in mc.
3) do we really need "-1" as a "disabled" value? As for me it is better not to have. If any node will decide to "disable" this feature, it may set up extremely big value.
4) field type must be `Long` since it is specified in zennies, not in zen. And add the comment to the config file and here that it consider zennies.